doc/jsk_topic_tools/lib/passthrough_nodelet.md

```diff
@@ -0,0 +1,54 @@
+# Passthrough
+
+`jsk_topic_tools/Passthrough` is a node/nodelet to relay topics only for specified duration. You can use the service call to turn relay on and off. By default, the topic relay is turned off.
+
+## Subscribing Topics
+
+* `~input` (`AnyMsg`)
+
+  Incoming topic to be relayed
+
+## Publishing Topics
+
+* `~output` (`AnyMsg`, same type as `~input`)
+
+  Outgoing topic to publish on
+
+## Services
+
+* `~request` (`std_srvs/Empty`)
+
+  Start topic relay
+
+* `~stop` (`std_srvs/Empty`)
+
+  Stop topic relay
+
+* `~request_duration` (`jsk_topic_tools/PassthroughDuration`)
+
+  Perform topic relay for a specified duration
```

Comment:
Could you add information like `Duration 0 means infinite relay`?
https://github.com/jsk-ros-pkg/jsk_common/blob/2.2.11/jsk_topic_tools/src/passthrough_nodelet.cpp#L88

Reply:
Thank you for your advice. I updated.

Reply:
Thank you!